src/lib/utils.test.ts

```diff
@@ -512,7 +510,7 @@ describe('utils', () => {
           '0x3a47ab5bd3a594c3a8995f8fa58d087600000000000000000000000000000000',
         ],
         values: [
-          '0x0000000000000000000000000000000000000000000000000000000000000001',
+          '0x00000000000000000000000000000001',
```

Comment:
Can you ad test with values that are wrong? And expect it to throw?

Here we are in a very very happy path.

What if input is "hello world" ? What if input is "11" ?
Do we get `0x00...000011` ? Or should we get `0x00...0000C` ??

Add more test cases please